src/components/ElvaYBPractice2/MainComponent.tsx

```diff
@@ -0,0 +1,130 @@
+  0: '',
+  25: '',
+  50: '',
+  75: '',
+  100: '',
+}
+
+const MainComponentContainer = () => {
+  const [views, setViews] = useState(50)
+  const [toggle, setToggle] = useState(false)
+  return (
+    <MainComponent>
+      <PageViewPriceContainer>
+        {toggle === false ? (
+          views === 0 ? (
+            <PageViewContainer>{10}k PAGEVIEWS</PageViewContainer>
+          ) : views === 25 ? (
+            <PageViewContainer>{50}k PAGEVIEWS</PageViewContainer>
+          ) : views === 50 ? (
+            <PageViewContainer>{100}k PAGEVIEWS</PageViewContainer>
+          ) : views === 75 ? (
+            <PageViewContainer>{500}k PAGEVIEWS</PageViewContainer>
+          ) : (
+            <PageViewContainer>{1}M PAGEVIEWS</PageViewContainer>
+          )
+        ) : views === 0 ? (
+          <PageViewContainer>{10}k PAGEVIEWS</PageViewContainer>
+        ) : views === 25 ? (
+          <PageViewContainer>{50}k PAGEVIEWS</PageViewContainer>
+        ) : views === 50 ? (
+          <PageViewContainer>{100}k PAGEVIEWS</PageViewContainer>
+        ) : views === 75 ? (
+          <PageViewContainer>{500}k PAGEVIEWS</PageViewContainer>
+        ) : (
+          <PageViewContainer>{1}M PAGEVIEWS</PageViewContainer>
+        )}
+        <PriceMonthContainer>
+          {toggle === false ? (
+            views === 0 ? (
+              <PriceContainer>$8.00</PriceContainer>
+            ) : views === 25 ? (
+              <PriceContainer>$12.00</PriceContainer>
+            ) : views === 50 ? (
+              <PriceContainer>$16.00</PriceContainer>
+            ) : views === 75 ? (
+              <PriceContainer>$24.00</PriceContainer>
+            ) : (
+              <PriceContainer>$36.00</PriceContainer>
+            )
+          ) : views === 0 ? (
+            <PriceContainer>$6.00</PriceContainer>
+          ) : views === 25 ? (
+            <PriceContainer>$9.00</PriceContainer>
+          ) : views === 50 ? (
+            <PriceContainer>$12.00</PriceContainer>
+          ) : views === 75 ? (
+            <PriceContainer>$18.00</PriceContainer>
+          ) : (
+            <PriceContainer>$27.00</PriceContainer>
+          )}
+          <MonthContainer>/month</MonthContainer>
+        </PriceMonthContainer>
+      </PageViewPriceContainer>
+      <Slidercontainer>
+        <StyleSlider
+          marks={marks}
+          onChange={(value: React.SetStateAction<number>) => setViews(value)}
+          step={null}
+          defaultValue={50}
+          handleStyle={{ backgroundColor: 'hsl(174, 86%, 45%)' }}
+          trackStyle={{ backgroundColor: 'hsl(174, 86%, 45%)' }}
+        />
+      </Slidercontainer>
+      <ToggleContainer>
+        <ToggleContainerText>Monthly Billing</ToggleContainerText>
+        <Switch onChange={(checked) => setToggle(checked)} />
+        <ToggleContainerText>Yearly Billing</ToggleContainerText>
+        <DiscountButton1>25% discount</DiscountButton1>
+        <DiscountButton2>25%</DiscountButton2>
```

Comment:
consider changing to more intuitive name eg. DiscountButtonMobile or DiscountButtonDesktop